src/cascadia/TerminalConnection/ConhostConnection.cpp

```diff
@@ -29,7 +29,8 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         _initialCols{ initialCols },
         _commandline{ commandline },
         _startingDirectory{ startingDirectory },
-        _guid{ initialGuid }
+        _guid{ initialGuid },
+        _connected{ false }

```

Comment:
@binarycrusader I may have gotten the outcome of the "where do I set values on construct?" discussion backwards. :smile: